backend/package.json

```diff
@@ -11,7 +11,7 @@
     "postbuild": "npm run copy-assets",
     "start": "node build/server",
     "copy-assets": "copyfiles -u 1 src/assets/* src/**/*.sql build",
-    "test": "jest",
+    "test": "jest --passWithNoTests",
```

Comment:
Added the flag `--passWithNoTests` because I moved the tests to the local module. We can remove this once we have new tests on `backend`.